documentation/docs/release-notes/3.0.0_GA.md

```diff
@@ -14,11 +14,11 @@ PMM 3.0.0 introduces this and that
 
 ## Release highlights
 
-### 
-
 ## Improvements
 
-- [PMM-xxx](https://perconadev.atlassian.net/browse/PMM-xxx) - 
+- [PMM-12644](https://perconadev.atlassian.net/browse/PMM-12644) - We have enhanced PMM's container management by transitioning from data containers to Docker volumes to store data in the `/srv` directory. This means your data is no longer locked to specific container images and you can more easily manage storage with the `docker volume ls` command. 
```

Comment:
```suggestion
- [PMM-12644](https://perconadev.atlassian.net/browse/PMM-12644) - We have enhanced PMM's container management by transitioning from data containers to Docker volumes to store data in the `/srv` directory. This means your data is no longer locked to specific container images and you can more easily manage storage with the `docker volume` command. 
```

Reply:
`ls` is just one subcommand, but there are more available to manage the volumes.